libs/partners/ai21/README.md

```diff
@@ -102,4 +102,18 @@ chain = tsm | StrOutputParser()
 response = chain.invoke(
     {"context": "Your context", "question": "Your question"},
 )
+```
+
+## Text Splitters
```

Comment:
I think its worth adding a link to the notebook you created which has more examples